diff --git a/esm/vs/base/browser/dom.js b/esm/vs/base/browser/dom.js
index b2acc4b70e172ec1f834649c5251f06ce612a657..d7a8a36fe2af93b172b0311e85d1257e0cdb8e62 100644
--- a/esm/vs/base/browser/dom.js
+++ b/esm/vs/base/browser/dom.js
@@ -9,7 +9,7 @@ import { StandardMouseEvent } from './mouseEvent.js';
 import { AbstractIdleValue, IntervalTimer, _runWhenIdle } from '../common/async.js';
 import { onUnexpectedError } from '../common/errors.js';
 import * as event from '../common/event.js';
-import * as dompurify from './dompurify/dompurify.js';
+import dompurify from 'dompurify';
 import { Disposable, DisposableStore, toDisposable } from '../common/lifecycle.js';
 import { FileAccess, RemoteAuthorities } from '../common/network.js';
 import * as platform from '../common/platform.js';
diff --git a/esm/vs/base/browser/markdownRenderer.js b/esm/vs/base/browser/markdownRenderer.js
index ceb5bb6eb94d0312360f1c1fc3db8e14e0dc6037..fe210f31bcc83cf9b3832500f2567961f7ad0c9a 100644
--- a/esm/vs/base/browser/markdownRenderer.js
+++ b/esm/vs/base/browser/markdownRenderer.js
@@ -3,7 +3,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 import * as DOM from './dom.js';
-import * as dompurify from './dompurify/dompurify.js';
+import dompurify from 'dompurify';
 import { DomEmitter } from './event.js';
 import { createElement } from './formattedTextRenderer.js';
 import { StandardKeyboardEvent } from './keyboardEvent.js';
diff --git a/esm/vs/base/browser/ui/button/button.js b/esm/vs/base/browser/ui/button/button.js
index 954a61f9cf658fa0872c7002837b33500be4316d..7f0879fec1a95f554b06c8fab33626fed8af6d7c 100644
--- a/esm/vs/base/browser/ui/button/button.js
+++ b/esm/vs/base/browser/ui/button/button.js
@@ -1,5 +1,5 @@
 import { addDisposableListener, EventHelper, EventType, reset, trackFocus } from '../../dom.js';
-import { sanitize } from '../../dompurify/dompurify.js';
+import dompurify from 'dompurify';
 import { StandardKeyboardEvent } from '../../keyboardEvent.js';
 import { renderMarkdown, renderStringAsPlaintext } from '../../markdownRenderer.js';
 import { Gesture, EventType as TouchEventType } from '../../touch.js';
@@ -153,7 +153,7 @@ export class Button extends Disposable {
             const root = rendered.element.querySelector('p')?.innerHTML;
             if (root) {
                 // Only allow a very limited set of inline html tags
-                const sanitized = sanitize(root, { ADD_TAGS: ['b', 'i', 'u', 'code', 'span'], ALLOWED_ATTR: ['class'], RETURN_TRUSTED_TYPE: true });
+                const sanitized = dompurify.sanitize(root, { ADD_TAGS: ['b', 'i', 'u', 'code', 'span'], ALLOWED_ATTR: ['class'], RETURN_TRUSTED_TYPE: true });
                 labelElement.innerHTML = sanitized;
             }
             else {
